### PR TITLE
Added timeline for PyCon India 2020 tasks

### PR DESCRIPTION
**Pull request by raukadah**
It includes:
* Rough timeline of various events
* Different tasks and their status

Signed-off-by: Chandan Kumar (raukadah) <raukadah@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,3 +123,31 @@ Volunteer Lead: Amol Kahat
...
+* Schedule shall be released on 1st September 2020.
+* Talks will be presented on 2nd and 3rd October 2020.
+* CFP closes on 14th August 2020
+* Ticket sales going live on <>.
+* Ticket sales closing date on <>.
+
+### Below is the milestones of various tasks:
+| **Workgroup** | **Owner** | **Total Target** |**Target Date Wise** | **Status** |
+|---|---|---|---|---|
+|Sponsership|Chandan Kumar||31st July, 2020|-|Going Live|
```
Comment (sayanchowdhury) [inline README.md]: Sponsorship

File: README.md [hunk 2]
```diff
@@ -123,3 +123,31 @@ Volunteer Lead: Amol Kahat
...
+||Sayan|||||
+||Sanchit|||||
+||Abhishek|||||
+|Reviewer|Ritesh|50|* 30th June, 2020 -> 15 | -|
```
Comment (sayanchowdhury) [inline README.md]: Let's not write the name that will be redundant information.

File: README.md [hunk 3]
```diff
@@ -123,3 +123,31 @@ Volunteer Lead: Amol Kahat
...
+### TLDR;
+* Sponsership going live on 31st July, 2020.
+* Schedule shall be released on 1st September 2020.
+* Talks will be presented on 2nd and 3rd October 2020.
+* CFP closes on 14th August 2020
+* Ticket sales going live on <>.
+* Ticket sales closing date on <>.
+
+### Below is the milestones of various tasks:
+| **Workgroup** | **Owner** | **Total Target** |**Target Date Wise** | **Status** |
```
Comment (sayanchowdhury) [inline README.md]: Add a field for just the Deadline and another field as Objective.

**Review by sayanchowdhury — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (raukadah) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (raukadah) [reply README.md]: Yes, removed

File: README.md (hunk 3 above)
Comment (raukadah) [reply README.md]: Now, we have 3 fields:
* workgroup
* Objective
* Deadline

Do we have objective defined somewhere so that we can add it here

**Review by raukadah — COMMENTED** [review]
Added comments

File: README.md (hunk 3 above)
Comment (sayanchowdhury) [reply README.md]: Objecttive will be targets for CFP, Mentorship workgroup. Maybe sometime else for other work groups

File: README.md (hunk 2 above)
Comment (sayanchowdhury) [reply README.md]: The deadline will just have the dates, and Objective will have the numbers.

**Comment by sayanchowdhury** [comment]
@raukadah Can you update the PRs?

**Review by raukadah — COMMENTED** [review]
Done

**Review by sayanchowdhury — APPROVED** [review]

**Pull request merged**